### DISCUSSION TITLE
Reality  problem with MCI(مشکل Reality در همراه اول)

### DISCUSSION THREAD

**Discussion by abas002** [XTLS/Xray-core#2178] [General]
hi
i use hiddify and this config: 
Reality+vless+grpc
but have problem with MCI and dont work.
who can help me?
thanks

**Comment by thenorthatlantic** [comment]
Hello, it appears that your server's IP address may have been blocked by MCI.

**Comment by vahidx4r4x** [comment]
سلام 
رفیق از gRPC روی همراه اول استفاده نکن چون کلاً پروتوکل gRPC روش بسته‌س
از rprx-vision روی tcp استفاده کن و پورت روی روی 443 بزار

**Reply by elegance666** [reply]
روی microsoft azure کار میکنه reality با gRPC روی همراه اول

**Reply by amir-devman** [reply]
Microsoft Azure asan mahdoodiat roosh nist aziz, hame chi roosh kar mikone
